infra/modules/worklytics-connectors-msft-365/main.tf

```diff
@@ -68,7 +68,7 @@ resource "local_file" "todo-with-external-todo" {
   for_each = local.todos_to_populate
 
   filename = module.msft_365_grants[each.key].filename
-  content  = format("%s\n## Setup\nThen, please follow next instructions for complete the setup: \n\n%s", module.msft_365_grants[each.key].todo, each.value.external_token_todo)
+  content  = format("%s\n## Setup\nThen, please follow next instructions for complete the setup: \n\n%s", module.msft_365_grants[each.key].todo, replace(each.value.external_token_todo, "%%entraid.application_id%%", module.msft_connection[each.key].connector.application_id))
```

Comment:
File is well generated, but every time that I run `terraform plan/apply` this is updated. Any ideas how I can enforce this to avoid that @eschultink ?

Reply:
commented below - but I don't see repeated changes on this on successive terraform applies